drizztsaga/language/french/setup.tra

```diff
@@ -16,7 +16,7 @@
 @353 = ~TUTU d�tect�, installation commenc�e...~
 @354 = ~BGT d�tect�, installation commenc�e...~
 @355 = ~BG1 original d�tect�, installation commenc�e...~
-@356 = ~ce mod doit �tre install sur BGEE, BGT ou BG TUTU, et Totsc et ToB sont requis pour Tutu/BGT...~
+@356 = ~ce mod doit �tre install sur BGEE, BGT ou BG TUTU, et Totsc et ToB sont requis pour Tutu/BGT...~
```

Comment:
Seems like auto conversion done by text editor, to revert?

Reply:
This file has been malformed since the initial upload and is broken even in the old v3 zip archive at BWL Downloads. I've reverted this via 18d5acf

BTW: Do you have to know where I can download v2 zip?

Reply:
I think it was from SHS selection box. But SHS is done right now...
I uploaded v1 and v2 in the source control so you can checkout to that tag if you need, Maybe you will find proper files.

Reply:
That would be perfect. At least you could verify the setup file, as it appears to be corrupted.